README.md

```diff
@@ -26,57 +26,43 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Webpack <a name="about-project"></a>
+# 📖 Free Games Giveaway <a name="about-project"></a>
 
-> Webpack is a boilerplate project used as a starting point for projects that requires webpack.
+> Free Games Giveaway is a website which you can find the latest links to giveaway game for multiple platforms : Windows, IOS, Android ...
 
 ## 🛠 Built With <a name="built-with"></a>
 - HTML
 - CSS
 - JS
 - Jest
+- Webpack
 ### Tech Stack <a name="tech-stack"></a>
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://www.w3schools.com/js/">Vanilla JS</a></li>
+    <li><a href="https://www.w3schools.com/js/">JavaScript</a></li>
     <li><a href="https://www.w3schools.com/html/">HTML</a></li>
     <li><a href="https://www.w3schools.com/css/">CSS</a></li>
   </ul>
 </details>
 
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Using the project will help you to : 
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Access to the latest giveaways**
+- **See people's insights**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://wassimchakib.github.io/Free-Games-Tracker/dist/)
 
```

Comment:
[OPTIONAL] The link to the live demo is returning error, i think it should be https://wassimchakib.github.io/Free-Games-Tracker/ since you used gh-pages deploying